docs/release-notes/v1.3.rst

```diff
@@ -1,7 +1,221 @@
+.. include:: ../references.txt
+
 .. _gammapy_1p3_release:
 
-1.3 (unreleased)
-----------------
+1.3 (October xx, 2024)
+-------------------------
+
+- Released October xx, 2024
+- xx contributors
+- 160 pull requests since v1.2 (not all listed below)
+- 59 closed issues
+
+Contributors
+~~~~~~~~~~~~
+- Stefan Fröse
+- Maxime Regeard
+- Bruno Khélifi
+- Claudio Galelli
+- Arnau Aguasca-Cabot
+- Fabio PINTORE
+- Daniel Morcuende
+- Atreyee Sinha
+- Brigitta Sipőcz
+- Michele Peresano
+- Maximilian Linhoff
+- Hanna Stapel
+- Pei Yu
+- AOI
+- stahanna
+- Quentin Remy
+- Kirsty Feijen
+- Fabio Acero
+- Régis Terrier
+- Lars Mohrmann
+- VascoSch92
```

Comment:
We also need this one!

Reply:
@VascoSch92 Can you please update your full name here, and for the future, ensure your git credentials are properly configured (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)?